@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -40,15 +44,30 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
     initialState: () => ({
       container: React.createRef(),
       rfb: undefined,
+      rfbConnected: false,
     }),
     effects: {
-      initialize: async function () {
+      initialize: function () {
+        this.effects._connect()
+      },
+      _handleDisconnect: function () {
+        this.state.rfbConnected = false
+        setTimeout(() => {
+          this.effects._connect()
+        }, 1000)
```

Comment:
if `_connect` throws, it will be stuck in the "Trying to reconnect" state forever.